pywt/_extensions/_pywt.pyx

```diff
@@ -233,16 +241,30 @@ def families(int short=True):
     --------
     >>> import pywt
     >>> pywt.families()
-    ['haar', 'db', 'sym', 'coif', 'bior', 'rbio', 'dmey']
+    ['haar', 'db', 'sym', 'coif', 'bior', 'rbio', 'dmey', 'gaus', 'mexh', 'morl', 'cgau', 'shan', 'fbsp', 'cmor']
     >>> pywt.families(short=False)
-    ['Haar', 'Daubechies', 'Symlets', 'Coiflets', 'Biorthogonal', 'Reverse biorthogonal', 'Discrete Meyer (FIR Approximation)']
+    ['Haar', 'Daubechies', 'Symlets', 'Coiflets', 'Biorthogonal', 'Reverse biorthogonal', 'Discrete Meyer (FIR Approximation)', 'Gaussian', 'Mexican hat wavelet', 'Morlet wavelet', 'Complex Gaussian wavelets', 'Shannon wavelets', 'Frequency B-Spline wavelets', 'Complex Morlet wavelets']
 
     """
     if short:
         return __wfamily_list_short[:]
     return __wfamily_list_long[:]
 
 
+def DiscreteContinuousWavelet(name=u"", object filter_bank=None):
```

Comment:
Agree that we can't really rename ``Wavelet`` without affecting a lot of user code. So okay with me to leave as is.